src/bolognese-sauce.md

```diff
@@ -1,5 +1,7 @@
 # Bolognese Sauce
 
+![Bolognese sauce](pix/bolognese-sauce-1.webp)
```

Comment:
Are there multiple images? If not, the image should be called `bolognese-sauce.webp` instead.